spec/system/user/user_settings_response_templates_spec.rb

```diff
@@ -20,8 +20,15 @@
 
       it "shows the proper message when deleting a reponse template", js: true do
         visit "/settings/response-templates"
-        accept_confirm { click_button "Remove" }
+        expect(page).to have_text(response_template.title)
 
+        expect(page).to have_css(".github-repo-row-name", text: response_template.title)
+        begin
+          accept_confirm { click_button("Remove") }
+        rescue Capybara::ModalNotFound => e
+          puts "This spec is known to his this error because " \
```

Comment:
```suggestion
          puts "This spec is known to hit this error because " \
```

Reply:
This PR description is really great, I would drop the PR number here or a link so people can find all the information and details.

Reply:
thanks for committing the change 👍